--- cairo-1.12.2/build/aclocal.cairo.m4.orig	2012-07-16 17:37:03.561908835 +0400
+++ cairo-1.12.2/build/aclocal.cairo.m4	2012-07-16 17:38:14.075821682 +0400
@@ -106,9 +106,9 @@
 		[cairo_cc_stderr=`test -f conftest.err && cat conftest.err`
 		 cairo_cc_flag=no])
 
-	if test "x$cairo_cc_stderr" != "x"; then
-		cairo_cc_flag=no
-	fi
+#	if test "x$cairo_cc_stderr" != "x"; then
+#		cairo_cc_flag=no
+#	fi
 
 	if test "x$cairo_cc_flag" = "xyes"; then
 		ifelse([$3], , :, [$3])
